docs/faq.rst

```diff
@@ -42,6 +42,23 @@ tox 4 - removed tox.ini keys
 | ``isolated_build``       | Isolated builds are now always used.        |
 +--------------------------+---------------------------------------------+
 
+tox 4 - changed behaviour for tox.ini keys
+++++++++++++++++++++++++++++++++++++++++++
+
+- When using any of the ``commands``, ``commands_pre`` or ``commands_post`` keys, you need to escape the ``#`` sign.
```

Comment:
Remove reference to commands, this is a general ini file behavior; these commands is just an example.